Sources/LiftBottomScrollView.swift

```diff
@@ -0,0 +1,52 @@
+import UIKit
+
+class LiftBottomScrollView: UIScrollView {
```

Comment:
I think you can remove the `Lift` suffix in internal classes since the context of usage is quite clear.